common-controller/cmd/main.go

```diff
@@ -30,6 +31,11 @@ func main() {
 	logger.Info("Starting the Web server")
 	go web.StartWebServer()
 	go server.StartInternalServer()
+	if conf.CommonController.Database.Enabled {
```

Comment:
Can this be nil? CommonController.Database

Reply:
No, it will be handled by default configs